probes/testsRunInCI/impl.go

```diff
@@ -165,7 +165,7 @@ func isTest(s string) bool {
 	for _, pattern := range []string{
 		"appveyor", "buildkite", "circleci", "e2e", "github-actions", "jenkins",
 		"mergeable", "packit-as-a-service", "semaphoreci", "test", "travis-ci",
-		"flutter-dashboard", "Cirrus CI", "azure-pipelines",
+		"flutter-dashboard", "Cirrus CI", "azure-pipelines", "woodpecker",
```

Comment:
Can we make this more specific to cut down on false positives? Based on the test data, does would `"ci/woodpecker"` work here?

Reply:
Well yes as this is the default string:

https://github.com/woodpecker-ci/woodpecker/blob/b1cf44ae0f80ca18dff3ab67b61cda59107b5a39/cmd/server/flags.go#L231